src/hal/src/lib.rs

```diff
@@ -631,4 +662,8 @@ pub trait Backend: 'static + Sized + Eq + Clone + Hash + fmt::Debug + Any + Send
     type Event: fmt::Debug + Any + Send + Sync;
     /// The corresponding query pool type for this backend.
     type QueryPool: fmt::Debug + Any + Send + Sync;
+
+    /// The corresponding acceleration structure type for this backend.
+    // TODO: This probably corresponds to `()` on DX12, since there's no handle for acceleration structures on DX12
```

Comment:
I thought it would be a buffer on DX12?

Reply:
You're absolutely right, I think this was a 3am brain fart